robottelo/performance/graph.py

```diff
@@ -113,3 +67,101 @@ def generate_line_chart_raw(time_result_dict, head, filename):
         time_list = time_result_dict.get(key)
         line_chart.add('client-{}'.format(thread), time_list)
     line_chart.render_to_file(filename)
+
+
+def generate_line_chart_raw_Pulp(time_result_dict, head, filename):
+    """Generate Normal Line chart for raw data of sync/resync"""
+    line_chart = pygal.Line()
+    generate_line_chart_raw(time_result_dict, head, filename, line_chart)
+
+
+def generate_line_chart_raw_Candlepin(time_result_dict, head, filename):
+    """Generate Normal Line chart for raw data of ak/att/del/reg"""
+    line_chart = pygal.Line(show_dots=False, range=(0, 50))
```

Comment:
fixed range again :(
could you calculate the `max_label` as you did at line [#80](https://github.com/SatelliteQE/robottelo/pull/2657/files#diff-0065083de42132eb44a84463371d1320R80)?